cleanup-passes/travis.ts

```diff
@@ -20,6 +20,21 @@ import * as path from 'path';
 import {ElementRepo} from '../element-repo';
 import {existsSync, makeCommit} from './util';
 
+type TravisEnv = {global?: string[], matrix?: string[]};
```

Comment:
I tried making this `string[] | {global?: string[]}`, but L136 would never be happy about that, insisting that `global` is not a property of `string[]`